features.json

```diff
@@ -229,14 +229,15 @@
 				"bigInt": null,
 				"bulkMemory": "1.0.30",
 				"exceptions": ["flag", "Requires flag `--enable-exceptions`"],
+				"extendedConst": ["flag", "Requires flag `--enable-extended-const`"],
 				"memory64": ["flag", "Requires flag `--enable-memory64`"],
 				"multiMemory": ["flag", "Requires flag `--enable-multi-memory`"],
 				"multiValue": "1.0.24",
 				"mutableGlobals": "1.0.1",
 				"referenceTypes": "1.0.31",
 				"saturatedFloatToInt": "1.0.24",
 				"signExtensions": "1.0.24",
-				"simd": false
+				"simd": "1.0.33"
```

Comment:
Does it not require a flag?

Reply:
It doesn't require a flag (SIMD was enabled by default in WABT shortly after it went to phase 5). But, wasm2c only puts SIMD support in the generated output if the module actually uses it, since it's a ton of code...